packages/trip-details/src/index.tsx

```diff
@@ -1,5 +1,4 @@
 import flatten from "flat";
-// @ts-expect-error FIXME: Create TypeScript types for core-utils packages.
```

Comment:
You still need this exception, this the change that is breaking the build.

Reply:
Moving to `ts-ignore` in 6783ac03e75892e3319d7d755759b836fac5cbda seems to fix things!